fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -1061,6 +1076,12 @@ ACTOR Future<bool> checkExclusion(Database db,
 		return false;
 	}
 
+	// If the exclusion command only contains processes that serve a non storage role we can skip the free capacity check in order to not
+	// block those exclusions.
```

Comment:
I think this is a good change. Maybe @jzhou77 knows why this code was assuming exclusion would always involve storage servers.